app/src/main/java/com/github/fribourgsdp/radio/GameActivity.kt

```diff
@@ -40,17 +52,22 @@ class GameActivity : AppCompatActivity(), GameView {
             val hostGameHandler = HostGameHandler(game, this)
             hostGameHandler.linkToDatabase()
             user = User("The best player")
-            playerGameHandler = PlayerGameHandler(game.id, this)
-        } else {
-            playerGameHandler = PlayerGameHandler(intent.getLongExtra(GAME_UID_KEY, -1L), this)
         }
+        playerGameHandler = PlayerGameHandler(intent.getLongExtra(GAME_UID_KEY, -1L), this)
```

Comment:
Here and at line 46 you both times get the extra id of the intent. It should be better to only get it once and store it in a `val`. Also, the default shouldn't be 0 as it is a possible lobby/game value, so it's better to set it to minus one.

Reply:
Good catch -> done!